data-catalog-backend-app/src/main/java/no/nav/data/catalog/backend/app/codelist/CodelistService.java

```diff
@@ -40,18 +41,33 @@ public void refreshCache() {
 		});
 	}
 
-	public Codelist save(CodelistRequest request) {
-		codelists.get(request.getList()).put(request.getCode(), request.getDescription());
-		return repository.save(request.convert());
+	public List<Codelist> save(List<CodelistRequest> requests) {
+		requests.forEach(request -> codelists.get(request.getList())
+				.put(request.getCode().toUpperCase(), request.getDescription()));
```

Comment:
Trim is very important here so that we do not add any whitespaces with data. Use both toUpperCase and Trim on all the the places. 
`.put(request.getCode().toUpperCase().trim(), request.getDescription()));`

Reply:
Added trim :)